### ISSUE TITLE
Can't run `foundation-apps watch` command after application creation

### ISSUE THREAD

**Issue by DevanB** [zurb/foundation-apps#586]
After installing the Foundation-CLI, running `foundation-apps new magnolia-lite`, changing into the magnolia-lite directory, and running `foundation-apps watch` I am getting an error.

Current environment:
Mac OS X 10.10.3, Node v0.12.2, NPM 2.7.5, Foundation CLI version 1.1.0 

Full error:
```
➜ magnolia-lite     foundation-apps watch

> foundation-apps-template@1.1.0 start /Users/devanb/Desktop/magnolia-lite
> gulp ""

[10:08:28] Using gulpfile ~/Desktop/magnolia-lite/gulpfile.js
Error: foundation-apps-template@1.1.0 start: `gulp ""`
Exit status 1
    at EventEmitter.<anonymous> (/usr/local/lib/node_modules/foundation-cli/node_modules/npm/lib/utils/lifecycle.js:213:16)
    at EventEmitter.emit (events.js:110:17)
    at ChildProcess.<anonymous> (/usr/local/lib/node_modules/foundation-cli/node_modules/npm/lib/utils/spawn.js:14:12)
    at ChildProcess.emit (events.js:110:17)
    at maybeClose (child_process.js:1015:16)
    at Process.ChildProcess._handle.onexit (child_process.js:1087:5)
```

**Comment by gakimball** [comment]
Can you try running `npm start` and see if that works?

**Comment by DevanB** [comment]
`npm start` does work and is what I've been using. Is the Zurb documentation incorrect and I should be using npm start instead?

**Comment by gakimball** [comment]
`npm start` is what the CLI runs in the background, so until we get this fixed you can use that command and it'll work just fine.

What versions of Node and npm are you running? Use `node -v` and `npm -v` to find out.

**Comment by gakimball** [comment]
@quook What version of Node are you on? Maybe it's an issue with 0.12; most of us here are still running 0.10.

**Comment by DevanB** [comment]
Thanks for the information @gakimball 

I'll leave the issue in your all hands then.

**Comment by quook** [comment]
@gakimball Node v0.12.2 (new machine, fresh install so latest version of everything). I can run the gulpfile directly with no trouble.

**Comment by heysailor** [comment]
Same error for me, node 0.12.2

**Comment by gakimball** [comment]
Alright, I was able to recreate it in Node 0.12.2, so it's an issue with Node 0.12.x.

**Comment by gakimball** [comment]
Just published [version 1.1](https://github.com/zurb/foundation-cli/releases/tag/v1.1.1) of the CLI which addresses this issue. Run `npm update -g foundation-cli` to get it.
